data/tutorial/04_contents.md

```diff
@@ -208,30 +208,30 @@ are used to implement JSON contents for Irmin.
 end
 ```
 
-## LWW register
+## LWW Register
 
-A last-write-wins register is similar to a basic Irmin store, except on merge
+A last-write-wins (LWW)) register is similar to a basic Irmin store; except on merge,
 the most recently written value will be picked rather than trying to merge the
 values.
 
-First, this requires a way to get a timestamp -- we will make this as generic as
-possible so it can be used on Unix or MirageOS:
+First, this requires a way to get a timestamp. We will make this as generic as
+possible, so it can be used on Unix or MirageOS:
 
 ```ocaml
 module type TIMESTAMP = sig
     val now: unit -> int64
 end
 ```
 
-On Unix this can be implemented `Unix.gettimeofday`:
+On Unix this can be implemented using `Unix.gettimeofday`:
 
 ```ocaml
 module Timestamp = struct
     let now () = Int64.of_float @@ Unix.gettimeofday () *. 100000.
 end
 ```
 
-`Lww_register` will be defined a functor that wraps an existing content type:
+`Lww_register` will be defined as a functor that wraps an existing content type:
```

Comment:
Should this be:
`Lww_register` will define a functor...
`Lww_register` will be defined a a functor...
??